src/hub/dataload/sources/dbnsfp/dbnsfp_mapping_48a_v1.py

```diff
@@ -0,0 +1,997 @@
+                }
+            },
+            # "integrated": {
+            #     "properties": {
+            #         "fitcons_score": { "type": "float" },
+            #         "fitcons_rankscore": { "type": "float" },
+            #         "confidence_value": { "type": "integer" }
+            #     }
+            # },
+            # "gm12878": {
+            #     "properties": {
+            #         "fitcons_score": { "type": "float" },
+            #         "fitcons_rankscore": { "type": "float" },
+            #         "confidence_value": { "type": "integer" }
+            #     }
+            # },
+            # "h1-hesc": {
+            #     "properties": {
+            #         "fitcons_score": { "type": "float" },
+            #         "fitcons_rankscore": { "type": "float" },
+            #         "confidence_value": { "type": "integer" }
+            #     }
+            # },
+            # "huvec": {
+            #     "properties": {
+            #         "fitcons_score": { "type": "float" },
+            #         "fitcons_rankscore": { "type": "float" },
+            #         "confidence_value": { "type": "integer" }
+            #     }
+            # },
+            "fitcons": {
+                "properties": {
+                    "integrated": {
+                        "properties": {
+                            **score_field,
+                            **rankscore_field,
+                            **confidence_value_field
+                        }
+                    },
+                    "gm12878": {
+                        "properties": {
+                            **score_field,
+                            **rankscore_field,
+                            **confidence_value_field
+                        }
+                    },
+                    "h1-hesc": {
+                        "properties": {
+                            **score_field,
+                            **rankscore_field,
+                            **confidence_value_field
+                        }
+                    },
+                    "huvec": {
+                        "properties": {
+                            **score_field,
+                            **rankscore_field,
+                            **confidence_value_field
+                        }
+                    },
+                }
+            },
+            "linsight": {
+                "properties": {
+                    **score_field,
+                    **rankscore_field
+                }
+            },
+            "gerp++": {
+                "properties": {
+                    "nr": {
+                        "type": "float"
+                    },
+                    "rs": {
+                        "type": "float"
+                    },
+                    "rs_rankscore": {
+                        "type": "float"
+                    }
+                }
+            },
+            "gerp": {
```

Comment:
@newgene, do you think the `gerp` field  is well structured in the mapping? It was also added to the mapping v2.

Reply:
not sure, have you checked against the actual data? If it matches the data, it should be fine.

Reply:
Yes I have. It's following the same pattern of other fields.